integration/simulation/stats/stats.go

```diff
@@ -56,8 +56,7 @@ func (s *Stats) L2Recalc(id common.Address) {
 
 func (s *Stats) NewBlock(b *types.Block) {
 	s.statsMu.Lock()
-	// s.l1Height = nodecommon.MaxInt(s.l1Height, b.Height)
-	s.TotalL1Blocks++
+	// s.l1Height = nodecommon.MaxInt(s.l1Height, b.Number)
```

Comment:
Do we mean to remove `s.TotalL1Blocks++` ?

Reply:
tbh, we should look at this stats thing again. I don't know how much sense it makes now. 
Added a comment here: https://github.com/obscuronet/obscuro-internal/issues/202